scripts/generateNftServiceKeypair.js

```diff
@@ -0,0 +1,89 @@
+import { KeyPair } from 'ucan-storage/keypair';
+import { build } from 'ucan-storage/ucan-storage';
+
+const SERVICE_ENDPOINT = 'https://api.nft.storage'; // default
+const API_TOKEN = process.env.API_KEY;
```

Comment:
was it discussed?

Reply:
what's exactly?

Reply:
API_TOKEN

Reply:
Yes, discussed, it is supposed to be there